cloud-azure/src/test/java/com/sequenceiq/cloudbreak/cloud/azure/AzureEncryptionResourcesTest.java

```diff
@@ -139,6 +142,40 @@ private void initActionFailedExceptionConversion() {
                 .thenAnswer(invocation -> new Retry.ActionFailedException(new CloudConnectorException(invocation.getArgument(0, Exception.class))));
     }
 
+    private static Object[][] getDesResourceId() {
+        return new Object[][]{
+                {"/subscriptions/dummySubscriptionId/resourceGroups/dummyAlphaResourceGroup/providers/" +
```

Comment:
Not a big deal but it is a helpful convention to add a comment just before the first `{arguments}` entry listing all the desired test method argument names. So, in this case, something like `// desResourceId resourceGroupName` (or adapted accordingly if adding or renaming arguments).